modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -1,40 +1,51 @@
-= Hardware and Software Requirements
+= Mule Runtime Engine Requirements for Hardware and Software
```

Comment:
Added the product name in the title for SEO power.

Reply:
we call it Mule Runtime or Mule everywhere else in the docs. I'm not sure about adding yet another term for it.

Reply:
Will fix in title & topic body.

Reply:
Actually, per Sanjna, first occurrence should be Mulesoft runtime engine, then can be Mule after that. I'll wait for the global change to do that though.